README.md

```diff
@@ -413,14 +433,16 @@ registering `eleventy-plugin-fluid` as follows:
 
 ```diff
 const fluidPlugin = require("eleventy-plugin-fluid");
-+ const markdownItEmoji = require("markdown-it-emoji");
 
 module.exports = function (config) {
 -    config.addPlugin(fluidPlugin);
 +    config.addPlugin(fluidPlugin, {
 +        markdown: {
 +            plugins: [
```

Comment:
The instructions on plugins block are little unclear to me. If the comment are omitted, the plugins block becomes an invalid array with ["markdown-it-emoji" ["markdown-it-anchor", {}],]